migtests/scripts/functions.sh

```diff
@@ -744,6 +745,14 @@ wait_for_exporter_event() {
 
         if [ $elapsed -ge $timeout_seconds ]; then
             echo "Timeout reached (${timeout_seconds}s). Proceeding without detecting sufficient ${exporter_db} events."
+            # Showing relevant log file for debugging
+            if [ "$exporter_db" == "source" ]; then
+                echo "Showing ${EXPORT_DIR}/logs/yb-voyager-export-data.log"
+                tail_log_file "yb-voyager-export-data.log" 2>/dev/null || echo "Log file not found or not readable"
+            else 
+                echo "Showing ${EXPORT_DIR}/logs/yb-voyager-export-data-from-target.log"
+                tail_log_file "yb-voyager-export-data-from-target.log" 2>/dev/null || echo "Log file not found or not readable"
+            fi
```

Comment:
File not found is handled by `tail_log_file` itself.
Please don't make changes without understanding :)